services/gcp/container/secrets.yml

```diff
@@ -7,7 +7,7 @@ notes: >-
   By default, secrets are stored unencrypted, and anyone who can read a Secret can read its contents (its `data` field).
   The Secret contents are mounted inside pods as files in the file system. Someone with ability to gain access into a Pod
   may freely read the contents of the secret.
-mitigations:
+mitigation: >-
```

Comment:
```suggestion
mitigations:
```
Should be a list

Reply:
I adjusted `services/gcp/compute/instances.yml` to have mitigations (plural) instead